metron-interface/metron-config/src/app/sensors/sensor-parser-config/sensor-parser-config.component.ts

```diff
@@ -191,8 +192,8 @@ export class SensorParserConfigComponent implements OnInit {
 
   ngOnInit() {
     this.route.params.subscribe(params => {
-      let id = params['id'];
-      this.init(id);
+      this.paramsID = params['id'];
```

Comment:
Storing the id in a class property allows me to use it when evaluating whether or not to populate the parserConfig object with the timestampField property and default `timestamp` value.